.github/wiki/CodeMagic.md

```diff
@@ -0,0 +1,79 @@
+# CodeMagic
+
+Use the CodeMagic template to start a new project with CodeMagic as the CI/CD tool.
+
+## Workflows and Steps
+
+Out of the box, the CodeMagic Template has the following workflows and steps:
+
+| test                      | deploy_app_store                                        | deploy_staging                          | deploy_release_firebase                   |
+| ------------------------- | ------------------------------------------------------- | --------------------------------------- | ----------------------------------------- |
+| Install bundle            | Install bundle                                          | Install bundle                          | Install bundle                            |
+| Run CocoaPods install     | Run CocoaPods install                                   | Run CocoaPods install                   | Run CocoaPods install                     |
+| Fastlane - Build and Test | Fastlane - Build and Test                               | Fastlane - Build and Test               | Fastlane - Build and Test                 |
+| Fastlane - Clean Up Xcov  | Fastlane Match                                          | Fastlane Match                          | Fastlane Match                            |
+| Danger                    | Fastlane - Build and Upload Production App to App Store | Fastlane - Build and Upload Staging App | Fastlane: Build and Upload Production App |
+
+## Trigger Map
+
+| Workflow                | Trigger                 |
+| ----------------------- | ----------------------- |
+| test                    | Create or Update a PR   |
+| deploy_staging          | Push branch `develop`   |
+| deploy_release_firebase | Push branch `release/*` |
+| deploy_app_store        | Push branch `main`      |
+
+## Environment
+
+### Variables
+
+- MATCH_PASSWORD
+
+> The password is used to encrypt/decrypt the Match repository to store the distribution certificates and provisioning profiles.
+
+- MATCH_SSH_KEY
+
+> An SSH private key is used for cloning the Match repository that contains your distribution certificates and provisioning.
+
+- KEYCHAIN_PASSWORD
+
+> The password to access the keychain.
+
+- FIREBASE_CLI_TOKEN
+
+> Firebase token https://firebase.google.com/docs/cli#cli-ci-systems for uploading build to Firebase Distributions and Analytics.
```

Comment:
```suggestion
> [Firebase token](https://firebase.google.com/docs/cli#cli-ci-systems) for uploading build to Firebase Distributions and Analytics.
```
we can use hyperlink text instead of directly use the link as text.